rhel7/Dockerfile.pgo-installer.rhel7

```diff
@@ -0,0 +1,36 @@
+ARG BASEOS
+ARG BASEVER
+ARG PREFIX
+FROM ${PREFIX}/pgo-base:${BASEOS}-${BASEVER}
+
+LABEL name="pgo-installer" \
+    summary="Crunchy PostgreSQL Operator - Installer" \
+    description="Crunchy PostgreSQL Operator - Installer"
+
+# Kubernetes repository
+RUN echo $'[kubernetes] \n\
+name=Kubernetes \n\
+baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64 \n\
+enabled=1 \n\
+gpgcheck=1 \n\
+repo_gpgcheck=1 \n\
+gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg' > /etc/yum.repos.d/kubernetes.repo
+
+# REVIEW IN PR **** confirm with build team ***
+RUN yum install -y https://download.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+
+# which is used in the ansible roles to verify
+# used to install ssk-keyen and gen ssh keys
+RUN yum -y install \
+    --setopt=skip_missing_names_on_install=False \
+    kubectl \
+    ansible \
+    which
```

Comment:
Similar comments to the CentOS file

Reply:
missed this one - will fix

Reply:
@jkatz I tested adding the download into the longer yum install script and the build failed - Seems like those two installs need to be separate

Reply:
These changes break the order of operations in the container build. This ends up installing an incorrect version of ansible that is incompatible with the current ansible install.  Reverting the change to the centos7 docker file to have 

```
yum install epel-release

yum install \
    other-stuff
```

Reply:
Each `RUN` instruction is a new layer in the container image. You can combine multiple commands into one `RUN` using `&&`:
```dockerfile
RUN yum install epel-release \
 && yum install \
    other-stuff
```

Reply:
ah nice - ill update that